_releases/2023-04-12-1.8.0-released.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Crystal 1.8.0 is released!
+version: 1.8.0
+summary:
+thumbnail: +
+author:
+---
+
+We are delivering a new release with several bugfixes and improvements. Below we list the most important or interesting changes, without mentioning several bugfixes and smaller enhancements. For more details, visit the [changelog](https://github.com/crystal-lang/crystal/releases/tag/1.8.0). Breaking changes are marked with ⚠️.
+
+Pre-built packages are available on [GitHub Releases](https://github.com/crystal-lang/crystal/releases/tag/1.8.0) and our official distribution channels.
+See [crystal-lang.org/install](https://crystal-lang.org/install/) for installation instructions.
+
+## Stats
+
+In this release we included [XXX changes since the 1.7.3 release](https://github.com/crystal-lang/crystal/pulls?q=is%3Apr+milestone%3A1.7.8) by XX contributors. We thank all the effort put into improving the language! ❤️
```

Comment:
We thank all the contributors for all effort put into improving the language! ❤️